### PR TITLE
Adds Gas Mask Variety

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Adds three variants of the gas mask, replaces fire locker gas masks with a random pick between the three new variants, and gives science lockers and vendors one of the new variants in place of the old one.

image

## Why It's Good For The Game
Gas masks are a go to mask, because they both hide your identity AND can be used as internals. This doesn't remove the old gas masks, but makes them much harder to find, and replaces the most common sources with one of three new variants that makes trade offs between eachother, so there are less of the all powerful greytide masks everywhere

## Changelog
:cl:
add: three gas mask variants
tweak: fire locker gas masks are chosen randomly between these three masks
tweak: science lockers and vendors get the face revealing mask that can use internals
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/traumas.dm [hunk 1]
```diff
@@ -69,7 +69,8 @@ SUBSYSTEM_DEF(traumas)
 												 /obj/item/trash/cheesie)),
 
 					  "greytide"  = typecacheof(list(/obj/item/clothing/under/color/grey, /obj/item/melee/baton/cattleprod,
-												 /obj/item/twohanded/spear, /obj/item/clothing/mask/gas)),
+												 /obj/item/twohanded/spear, /obj/item/clothing/mask/gas, /obj/item/clothing/mask/gas/alt, /obj/item/clothing/mask/gas/alt2,
```
Comment (Fikou) [inline code/controllers/subsystem/traumas.dm]: they dont strike the same fear as the old one

Comment (Paxilmaniac) [reply code/controllers/subsystem/traumas.dm]: ok good point

**Comment by wesoda25** [comment]
I mean I guess

**Comment by cacogen** [comment]
I like the new? atmos gas mask, it looks menacing like New Vegas and is a refreshing change to that old tired sprite. I don't like these sprites though. If you just made recolours of the existing sprite in the same vein as that i.e. lenses and/or the gas mask itself that'd be cool

**Comment by itseasytosee** [comment]
Halp grey man with mask robust me?? I do not know who he is?? remobe bad mask!

**Comment by Paxilmaniac** [comment]
it doesn't remove the gas mask we already have, it just makes them harder to get than just "hmm yes red locker in hall"

**Comment by Twaticus** [comment]
did you mean to remove the atmos gas mask mob sprite or?

**Comment by Paxilmaniac** [comment]
in case it isn't clear: nothing is removed in this pr, it adds three masks that spawn in place of the current gas masks in the red lockers around the hallways.

this doesn't touch the atmos gas mask at all

**Comment by Paxilmaniac** [comment]
also the mob sprite for atmos gas masks shouldn't be removed?

File: code/game/machinery/gulag_teleporter.dm [hunk 2]
```diff
@@ -30,7 +30,9 @@ The console is located at computer/gulag_teleporter.dm
 		/obj/item/clothing/head/helmet/space/plasmaman,
 		/obj/item/tank/internals,
 		/obj/item/clothing/mask/breath,
-		/obj/item/clothing/mask/gas))
+		/obj/item/clothing/mask/gas,
+		/obj/item/clothing/mask/gas/alt,
+		/obj/item/clothing/mask/gas/alt2))
```
Comment (Jalleo) [inline code/game/machinery/gulag_teleporter.dm]: This could cause issues did you test this?

**Comment by trollbreeder** [comment]
I like these sprites, especially the shielded visor gas mask.

**Comment by Mickyan** [comment]
You need internals for fighting fires, would rather not have to play the rng lottery during an emergency

Personally I'd just swap them out for the regular breath mask but hey

**Comment by Paxilmaniac** [comment]
only one of the three cant use internals and its rare to get it, and you have a breath mask in your box, and there are some in every emergency locker, and emergency lockers happen to be almost always right next to a fire locker, so it shouldn't be a problem

**Comment by ExcessiveUseOfCobblestone** [comment]
image
?

**Comment by Paxilmaniac** [comment]
gas_atmos-2 was a mistake i accidentally put in which was literally an accidental copy paste of the atmos gas mask mob sprite with a rogue pixel, if you check the current mob mask sprites the atmos mask sprite is still there
image

REMINDER TO NOT CODE AT THREE AM OK??

**Comment by wesoda25** [comment]
Why do we need it though. Does anyone even complain about this stuff?

**Comment by Paxilmaniac** [comment]
i remember people talking about it a while back,

image

**Comment by wesoda25** [comment]
That comment doesn't even make sense though, how does it make sense to make gas masks, the thing you are supposed to have when fighting fire, not be able to be connected to internals? I don't even get whats "good" about the mask, its a liable reason to be arrested by shitsec so any real advantage it provides is ???? This PR seems like pointless bloat tbh

**Comment by JMoldy** [comment]
Can we have literally any gas mask actually function as a gas mask? It's just weird that NONE of the **four** gas masks on the station would be able to actually **work as a gas mask**.

**Comment by Paxilmaniac** [comment]
im going to make the 3rd and 4th gas masks actually look different from the original

**Comment by carlarctg** [comment]
conflict

the mask with  big visor looks really goofy. like it looks dumb. like the big head looks bubbly and it dumb

**Comment by anconfuzedrock** [comment]
imagine getting a fucking mask that doesn't even let you breathe from a fire closet in an emergency. Why's the third actually exist? masks that conceal your identity but don't let you breathe are super easy- you can legit make a paper bag mask even. not that anything's hard especially on stations like meta and delta where there's a fuckin autodrobe in public.

**Comment by Paxilmaniac** [comment]
mega oopsie, and im too lazy to fix it

**Comment by Paxilmaniac** [comment]
actually get this i'm just going to stop because i seem to have completely fucked it up

**Pull request closed without merging**